.github/workflows/main.yml

```diff
@@ -0,0 +1,27 @@
+name: Release
+
+on:
+  push:
+    # Limit this action to tags only
+    tags:
+      - '*'
```

Comment:
Should we limit this to semantic version tags? `v*.*.*`?

Reply:
Added as a regex `v[0-9]+\.[0-9]+\.[0-9]+` Tested on private repo and it works